pkgs/development/python-modules/plotnine/default.nix

```diff
@@ -75,6 +80,9 @@ buildPythonPackage rec {
     "tests/test_facet_labelling.py"
     "tests/test_coords.py"
     "tests/test_annotation_logticks.py"
+
+    # Linting / formatting: useless as it has nothing to do with the package functionning
```

Comment:
does this test fail for some reason? let's default to running more tests rather than fewer, even if they do not seem directly relevant

Reply:
I guess they just unnecessarily pull in the linters.

Reply:
Yes, I clarified the comment.